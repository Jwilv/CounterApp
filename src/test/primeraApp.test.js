import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import PrimeraApp from "../PrimerApp"

describe('pruebas en primera app ', () => {


    /*

 test('se debe mostrar el mensaje hola mundo', () => { 

        const mensaje = 'Hola mundo';

        const { getByText } = render(<PrimeraApp saludo={ mensaje } />)

       expect( getByText(mensaje) ).toBeInTheDocument();
     })

    */

    test('debe  mostrar <primeraApp /> correctamente ', () => {
        const saludo = 'anachichi'
        const wrapper = shallow(<PrimeraApp saludo={saludo} />)

        expect(wrapper).toMatchSnapshot()
    })

    test('debe mostrar el subtitulo enviado por props', () => {
        const saludo = 'anachichi'
        const subTitulo = 'hola buenas'
        const wrapper = shallow(
            <PrimeraApp
                saludo={saludo}
                subtitulo={subTitulo}

            />
        );

        const textoParrafo = wrapper.find('p').text()
        console.log(textoParrafo)
        expect(textoParrafo).toBe(subTitulo)
    })


})