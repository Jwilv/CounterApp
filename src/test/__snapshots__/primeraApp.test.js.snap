// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en primera app  debe  mostrar <primeraApp /> correctamente  1`] = `
<Fragment>
  <h1>
    anachichi
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
